CONTRIBUTING.md

```diff
@@ -78,41 +148,155 @@ Help us keep this project open and inclusive. Please read and follow our [Code o
 
 - Test your changes (create new tests as needed)
 
-- Submit a Pull Request
+- Once you’ve committed and pushed all of your changes to GitHub, go to the page for your fork on GitHub, select your development branch, and click the pull request button. Please ensure that you compare your feature branch to the desired branch of the repo you are supposed to make a PR to. If you need to make any adjustments to your pull request, just push the updates to GitHub. Your pull request will automatically track the changes in your development branch and update it. 🥳
+
+## (Option 2) Working on an issue - Make the changes via Locally
+
+Make sure you have the following prerequisites installed on your operating system before you start contributing:
+
+- [Nodejs and npm](https://nodejs.org/en/)
+
+  To verify run:
+
+  ```
+  node -v
+  ```
+
+  ```
+  npm -v
+  ```
+
+### Set up your Local Development Environment
+
+Follow the following instructions to start contributing.
+
+**1.** Fork [this](https://github.com/BlackPythonDevs/blackpythondevs.github.io) repository.
+
+
+**2.** Clone your forked copy of the project.
+
+⚠️ Cloning this repository ⚠️
+
+Cloning the repo with all its history could result in large portion of space (GB) being occupied on your machine. If you don't need the whole history you can use the depth parameter to significantly reduce download size.
+
+```
+git clone --depth=1 https://github.com/<your-username>/BlackPythonDevs/blackpythondevs.github.io.git
+```
+
+**3.** Navigate to the project directory.
+
+```
+cd Black Python Devs
+```
+
+**4.** Add a reference(remote) to the original repository.
+
+```
+git remote add upstream https://github.com/BlackPythonDevs/blackpythondevs.io.git
+```
+
+**5.** Check the remotes for this repository.
+
+```
+git remote -v
+```
+
+**6.** Always take a pull from the upstream repository to your main branch to keep it at par with the main project (updated repository).
+
+```
+git pull upstream main
+```
+
+**7.** Create a new branch.
+
+```
+git checkout -b <your_branch_name>
+```
+
+**8.** Install the dependencies for running the site.
+
+```
+npm install
+```
+
+**9.** Make the desired changes.
+
+**10.** Run the site locally to preview changes.
+
+```
+npm run build
+```
```

Comment:
Same issue as previous comment this would  be jekyll and bundle commands

`bundle exec jekyll serve`